modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -20,7 +21,7 @@
 import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
 import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
 
-@RunWith(Parameterized.class)
+@RunWith(ParallelParameterized.class)
```

Comment:
`ParallelParameterized` from `junit-toolbox` lets us run these parameterised tests in parallel.
Before merging I'd like to have a bit of a discussion about whether this is a route we should take, though.

Reply:
I have no experience with `ParallelParameterized`, so no real opinion here :slightly_smiling_face: